core/src/main/java/com/oath/oak/Operation.java

```diff
@@ -1,18 +0,0 @@
-/*
```

Comment:
So sad, rebalance is farer and farer away from what it needs to be :(

Reply:
I don't understand this comment. How `Opetation.java` relate to rebalancing?